jobs/scalingengine/templates/pre-start.erb

```diff
@@ -29,15 +29,31 @@ USER='<%= role['name'] %>'
 PASSWORD='<%= role['password'] %>'
 PORT='<%= p("autoscaler.scalingengine_db.port") %>'
 SSLMODE='<%= p("autoscaler.scalingengine_db.sslmode") %>'
-JDBCDBURL="jdbc:postgresql://$HOST:$PORT/$DBNAME?sslmode=$SSLMODE"
-DBURI="postgres://$USER:$PASSWORD@$HOST:$PORT/$DBNAME?sslmode=$SSLMODE"
-
-<% unless p('autoscaler.scalingengine_db.tls.ca') == "" %>
-JDBCDBURL=$JDBCDBURL"&sslrootcert=/var/vcap/jobs/scalingengine/config/certs/scalingengine_db/ca.crt"
+<% if p('autoscaler.scalingengine_db.db_scheme') =="postgres" %>
+ JDBCDBURL="jdbc:postgresql://$HOST:$PORT/$DBNAME?sslmode=$SSLMODE"
+ DBURI="postgres://$USER:$PASSWORD@$HOST:$PORT/$DBNAME?sslmode=$SSLMODE"
+<% unless p('autoscaler.scalingengine_db.tls.ca') == "" %> 
+ JDBCDBURL=$JDBCDBURL"&sslrootcert=/var/vcap/jobs/scalingengine/config/certs/scalingengine_db/ca.crt"
+ DBURI=$DBURI"&sslrootcert=/var/vcap/jobs/scalingengine/config/certs/scalingengine_db/ca.crt"
+ <% end %>
+<% else %>
+JDBCDBURL="jdbc:mysql://$HOST:$PORT/$DBNAME?autoReconnect=true"
+DBURI="$USER:$PASSWORD@tcp($HOST:$PORT)/$DBNAME?tls=$SSLMODE"
+<%unless p('autoscaler.scalingengine_db.tls.ca') == "" %>
+/var/vcap/packages/common/init-db-truststore.sh scalingengine_db /var/vcap/jobs/scalingengine/config/certs/scalingengine_db/ca.crt 
+JDBCDBURL=$JDBCDBURL"&useSSL=true&requireSSL=true&verifyServerCertificate=true&trustCertificateKeyStorePassword=123456&trustCertificateKeyStoreUrl=file:/var/vcap/data/certs/scalingengine_db/cacerts&trustCertificateKeyStoreType=pkcs12"
 DBURI=$DBURI"&sslrootcert=/var/vcap/jobs/scalingengine/config/certs/scalingengine_db/ca.crt"
-<% end %>
+ <% end %> 
+ <% end %>
 
 detect_dns $HOST $PORT
 /var/vcap/packages/changeloglockcleaner/changeloglockcleaner -d "$DBURI" -e <%= p("autoscaler.changeloglock_timeout_seconds") %>
+<% if p('autoscaler.scalingengine_db.db_scheme') == "postgres" %>
 java -cp "/var/vcap/packages/db/target/lib/*" liquibase.integration.commandline.Main --url "$JDBCDBURL" --username=$USER --password=$PASSWORD \
 --driver=org.postgresql.Driver --changeLogFile=/var/vcap/packages/scalingengine/scalingengine.db.changelog.yml update
+<% else %>
+java -cp "/var/vcap/packages/db/target/lib/*" liquibase.integration.commandline.Main --url "$JDBCDBURL" --username=$USER --password=$PASSWORD \
+--driver=com.mysql.cj.jdbc.Driver --changeLogFile=/var/vcap/packages/scheduler/db/databasechangelog.yaml update
```

Comment:
why use `scheduler` here?

Reply:
I just select one job of instance group asactors to put the changelog file. Maybe create a new folder and move it to the folder?